replica/table.cc

```diff
@@ -2966,4 +2966,9 @@ bool table::requires_cleanup(const sstables::sstable_set& set) const {
     }));
 }
 
+future<> table::cleanup_tablet(locator::tablet_id) {
+    co_await flush();
+    // FIXME: Remove sstables
```

Comment:
this triggers a shard wide flush, will probably reduce it to tablet granularity. anyway, it's irrelevant until I get to it.